components/button/button.css

```diff
@@ -0,0 +1,73 @@
+body {
+  font-family: var(--nunito);
+}
+
+.solid-btn {
+  border: none;
+  border-radius: 0.4rem;
+}
+
+.outline-btn {
+  border: 2px solid;
+  border-radius: 0.4rem;
+  background-color: var(--white);
+}
+
+.icon-btn {
+  display: flex;
+  border-radius: 0.4rem;
+  border: none;
+  text-align: center;
+  line-height: var(--lh-s);
+  margin: unset;
+}
+
+.s {
+  padding: 0.3rem 0.5rem;
+}
+
+.m {
+  padding: 0.5rem 0.8rem;
+}
+
+.l {
+  padding: 0.8rem 1rem;
+}
```

Comment:
the name doesn't tell what that class is for, naming can be improved.

Reply:
changed to .btn-lg